lib/glueby/internal/wallet/active_record_wallet_adapter.rb

```diff
@@ -89,20 +89,30 @@ def balance(wallet_id, only_finalized = true)
           utxos.sum(&:value)
         end
 
+        # If label=nil, it will return unlabeled utxos to protect labeled utxos for specific purpose
+        # If label=:all, it will return all utxos
```

Comment:
Method documentation should be placed at `abstract_wallet_adapter.rb` where declares interfaces.

Reply:
Here should describe :unlabeled. It can be passed.